ui/app/helpers/message-types.js

```diff
@@ -20,8 +20,8 @@ const MESSAGE_TYPES = {
     text: 'Error',
   },
   warning: {
-    class: 'is-highlight',
```

Comment:
This breaks other places where we want `is-highlight`.

This is on the sealed page:
![screen shot 2018-11-27 at 9 32 24 am](https://user-images.githubusercontent.com/39469/49092296-5ba53480-f227-11e8-8ee3-da96cd0357d5.png)

And when you log in with a root token:
![screen shot 2018-11-27 at 9 32 51 am](https://user-images.githubusercontent.com/39469/49092334-724b8b80-f227-11e8-8080-e23d5f4cf42b.png)

I think there may be a couple of other places we want `is-highlight` too.

Reply:
These should be fixed if we switch the `&.is-highlight` selector to `&.is-warning` in messages.scss